plugin/src/main/java/io/fabric8/maven/plugin/converter/DeploymentOpenShiftConverter.java

```diff
@@ -99,7 +101,7 @@ public HasMetadata convert(HasMetadata item) {
                 specBuilder.addNewTrigger().withType("ConfigChange").endTrigger();
 
                 // add a new image change trigger for the build stream
-                if (containerToImageMap.size() != 0) {
+                if (containerToImageMap.size() != 0 && useImageStreamTriggers) {
```

Comment:
until we figure out when they have any actual use, I'd prefer we disable them ;)